src/status_im/contexts/communities/overview/view.cljs

```diff
@@ -365,7 +365,7 @@
        :type       :title
        :text-align :left
        :icon-name  :i/close
-       :on-press   #(rf/dispatch [:navigate-back])}]
+       :on-press   #(rf/dispatch [:pop-to-root :shell-stack])}]
```

Comment:
hi @jo-mut, Thank you for working on the issue.

Did you tried pressing hardware back button, instead of close button? I think we will be navigated back to community chat instead of home screen in that case.